src/StoreApi/Routes/Products.php

```diff
@@ -309,10 +309,14 @@ public function get_collection_params() {
 
 		$params['stock_status'] = array(
 			'description'       => __( 'Limit result set to products with specified stock status.', 'woo-gutenberg-products-block' ),
-			'type'              => 'string',
-			'enum'              => array_keys( wc_get_product_stock_status_options() ),
```

Comment:
Sorry if I wasn't clear yesterday. We should keep `enum` if possible, I just wasn't sure if it needs to be defined within or outside of items. Ideally we can keep this in place so invalid values are rejected.

Reply:
Changes pushed, please check again. I also moved `sanitize_callback` and `validate_callback` under items.